docs/source/transforms.rst

```diff
@@ -9,124 +9,304 @@ Functional transforms give fine-grained control over the transformations.
 This is useful if you have to build a more complex transformation pipeline
 (e.g. in the case of segmentation tasks).
 
+.. Note::
+    Most transform classese have an equivalent in :mod:`torchvision.transforms.functional`.
+
+
 .. autoclass:: Compose
 
 Transforms on PIL Image
 -----------------------
 
 .. autoclass:: CenterCrop
 
+**Example**
+
+.. plot::
+   :include-source:
+
+    from torchvision.transforms import CenterCrop
+    from torchvision.utils import _plot_images, _sample_image
+
+    _plot_images(
+        _sample_image(),
+        CenterCrop(256)(_sample_image()),
+        CenterCrop((200, 300))(_sample_image()),
+    )
+
+
 .. autoclass:: ColorJitter
 
+**Example**
+
+.. plot::
+   :include-source:
+
+    from torchvision.transforms import ColorJitter
+    from torchvision.utils import _plot_images, _sample_image
```

Comment:
I was thinking if it wouldn't be maybe better to have those helper functions be in the documentation folder, instead of torchvision?

I don't think we want the docs to show private functions, as they would in fact encourage people to use those private functions

Reply:
My thought is that it would be nice to be able to copy and paste the examples and go from there. If the two functions are only part of the docs folder then it's not possible to reproduce the example.

I'm not sure how to best add these function only to the sphinx build.

Reply:
Hum, I'm not sure what's the best way to go here.

Asking for feedback from @brianjo @soumith if we should have those helper functions in torchvision or not, given that they will be mostly used for illustration purposes.

Reply:
Ok, waiting for input here.

Reply:
pinging @jlin27 what do you think the above?